userspace/libsinsp/parsers.cpp

```diff
@@ -2247,6 +2253,64 @@ inline void sinsp_parser::add_socket(sinsp_evt *evt, int64_t fd, uint32_t domain
 	evt->m_fdinfo = evt->m_tinfo->add_fd(fd, &fdi);
 }
 
+/**
+ * If we receive a call to 'sendto()' and the event's m_fdinfo is nullptr,
+ * then we likely missed the call to 'socket()' that created the file
+ * descriptor.  In that case, we'll guess that it's a SOCK_DGRAM/UDP socket
+ * and create the fdinfo based on that.
+ *
+ * Preconditions: evt->m_fdinfo == nullptr and
+ *                evt->m_tinfo != nullptr
+ * 
+ */
+inline void sinsp_parser::infer_sendto_fdinfo(sinsp_evt* const evt)
+{
+	if((evt->m_fdinfo == nullptr) || (evt->m_tinfo != nullptr))
```

Comment:
Don't we need to invert the conditions here? i.e.

```
if(evt->m_fdinfo != nullptr || evt->m_tinfo == nullptr) ...
```

Reply:
Yep :man_facepalming: -- thanks